package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "vscode-openshift-connector",
   "displayName": "OpenShift Connector",
-  "description": "Interacting with Red Hat OpenShift clusters and providing a streamlined developer experience using Visual Studio Code",
+  "description": "Simplified app dev for Kubernetes or Red Hat OpenShift",
```

Comment:
```suggestion
  "description": "Simplified application development with Kubernetes and Red Hat OpenShift",
```

Reply:
Why you did not tell anything in the issue till now?

Reply:
I am pushing option 7, for the reasons I noted in the issue.

Reply:
I will discuss in the issue, but I do not think short forms justify the workflow of the extension.